encryptonize.go

```diff
@@ -430,3 +434,27 @@ func (e *Encryptonize) GetGroupData(authorizer *SealedUser, group *SealedGroup)
 
 	return plainGroup.Data, nil
 }
+
+////////////////////////////////////////////////////////
+//                       Index                        //
+////////////////////////////////////////////////////////
+
```

Comment:
We're missing a method for creating a new index. That would also be a good place to explain the feature a bit (in the doc string).

Reply:
Fixed